secrets-provider-chart/values.yaml

```diff
@@ -0,0 +1,28 @@
+# K8s permission defaults
+rbac:
+  roleName: secrets-provider-role
+  roleBindingName: secrets-provider-role-binding
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # The name of the service account to use.
+    name: secrets-provider-service-account
+
+secretsProvider:
+  image: cyberark/secrets-provider-for-k8s
+  tag: 1.0.0
```

Comment:
do we want this static?

Reply:
see my above response to the previous comment